feature_engine/wrappers/wrappers.py

```diff
@@ -357,15 +369,18 @@ def get_feature_names_out(self) -> List:
         if self.transformer_.__class__.__name__ in _TRANSFORMERS:
             feature_names = self.feature_names_in_
```

Comment:
we need to modify this behaviour based on input_features

Reply:
Are you envisioning changing the above code from `feature_names = self.feature_names_in_` to `feature_names=self.get_feature_names_out(input_features=input_feature)`?

It looks like the classes included in `_TRANSFORMERS` are scikit-learn transformers. However, not all of the transformers, e.g. OrdinalEncoder, do have a `get_feature_names_out()` method.

Do we check if the class uses `get_feature_names_out()`? If the method exists we apply the abovementioned revised code and, if not, we keep `feature_names = self.feature_names_in_`.